src/app.ts

```diff
@@ -1,33 +1,47 @@
 import express from 'express'
-import { JwtAuthorizationMiddleware } from './types'
+import Session from 'express-session'
 import { quoteRouter } from './routes/quote'
 import { kycRouter } from './routes/kyc'
 import { accountsRouter } from './routes/accounts'
 import { transferRouter } from './routes/transfer'
 import { errorToStatusCode } from './middleware/error'
+import { authRouter } from './routes/auth'
+
+// this is typically the name of the provider
+const SESSION_NAME = 'api-starter'
```

Comment:
if it's important for providers to fill this in I suggest we replace it with a getter function that throws an error unless the implementor has filled it in